### DISCUSSION TITLE
Which global demand data is recommended for use in 30 arcmin CWatM

### DISCUSSION THREAD

**Discussion by Zprivf** [iiasa/CWatM#67] [Q&A] [answered]
Hello @mikhailsmilovic ,

There are two global demand data for 30min model. 
The first data is from "cwatm_input30min/landsurface/waterDemand" folder monthly from Inge's folder (1960-2010). 
The second is from a folder in the FTP server inside "Water demand scenario". This data is from ISIMIP, annual, and also is with future demand predictions (1850-2018 and 2015-2100), it also included livestock demand which the first didn't. 
Which one is the recommended demand data to use for global modeling including past demand, if one? Which data was used for the validation of the global model in Burek et al. (2020) paper? 

Thank you!

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Morgen,

**Water demand in the global 30min repository has now been updated to use that from the ISIMIP3 runs. We recommend continuing with this for new runs: https://github.com/iiasa/CWatM-Earth-30min**
The settings template has also been updated.

The previous demand files from Burek et a. (2020) are available in the FTP server in waterdemand_scenarios/30arcmin/previous. 
Two things to note for the previous version (readme.txt now also includes this information):
- all units are in metres/day (compared to million metres cubed per year in the updated version), therefore 
in the settings file, in [WATERDEMAND], demand_unit = True
- livestock demand is monthly (compared with annually in the updated version), therefore 
in the settings file in [WATERDEMAND], livestockTimeMonthly = True



Thank you for highlighting this!

**Reply by Zprivf** [reply]
Thank you @mikhailsmilovic. This is very helpful.
